config/routes.rb

```diff
@@ -5,6 +5,10 @@
   resources :articles do
     resources :comments
   end
+
+  resources :articles do
+    resources :likes
```

Comment:
are we using all routes? if not use 'only' to avoid extra routes. Read about 'only' and 'except' in routes.